content/en/docs/Installation/packages.md

```diff
@@ -28,10 +28,11 @@ More packages available, with links to download/install instructions:
 |-------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Cloudron                | https://www.cloudron.io/store/org.navidrome.cloudronapp.html                                                                                                                  |
 | Fedora                  | https://copr.fedorainfracloud.org/coprs/lchh/navidrome/                                                                                                                       |    
-| Kubernets Chart         | https://artifacthub.io/packages/helm/emmas-charts/navidrome                                                                                                                   | 
+| Helm Chart              | https://artifacthub.io/packages/helm/emmas-charts/navidrome                                                                                                                   |
```

Comment:
This one has a really old version, 0.50.0. Is it maintained at all?

Reply:
I honestly don't think so, I've used TrueCharts and the Helm Chart so I know it works, is maintained

Reply:
Removed @deluan , let me know if anything else is needed!